packages/offix-datastore/src/Model.ts

```diff
@@ -89,4 +133,34 @@ export class Model<T = unknown> {
                 listener(event);
             });
     }
+
+    private async subscribeForServerEvents(replicator: IReplicator) {
```

Comment:
I like that we have it globally available, but WDYT if we will have it public predicate instead. 

This will mean that user can:

- Add predicate generically on init 
- Subscribe to server events in specific view only (without persisting this predicates or reatempting this on startup. 

Not sure if that makes sense for now, but would love if you can tell me what you think about this.